.github/workflows/ci.yml

```diff
@@ -59,31 +59,3 @@ jobs:
           elixir-version: ${{matrix.elixir}}
       - run: mix deps.get
       - run: mix credo
-
-  Dialyzer:
```

Comment:
If the concern is how long it takes to run maybe it is worth just adding `Dialyzer` to the `deploy.yml` workflow - not as a blocker to deployment necessarily, but as a warning if there is a dialzyer issue that makes it to `master`.

That is, unless there is another reason to remove it besides how long it takes to run.

Reply:
Updated!